app/assets/javascripts/mtl/configuration.coffee.erb

```diff
@@ -0,0 +1 @@
+@MTL.file_icons = <%= Mtl.file_icons.to_json %>
```

Comment:
sorry if I wasn't clear before, in JS we should make use of camelCasedMethodNames